app/jobs/api_updater.rb

```diff
@@ -0,0 +1,10 @@
+class ApiUpdater < Updater
+  attr_reader :updater
```

Comment:
`attr_reader` defines an accessor method for an instance variable (`@updater` in this case). If you just want to enforce that subclasses implement an `updater` method, I'd do something like 

```
def updater
  raise StandardError.new('Must implement #updater in subclass')
end
```